provisioning-workflows/README.md

```diff
@@ -1,2 +1,5 @@
 # provisioning-workflow
 This repository contains workflow definitions (XML files) and other associated material to support automated provisioning of middleware environment. 
+
+# Installation
+You can install any of these workflows into z/OSMF following the instructions about [create templates for z/OS software provisioning] (https://www.ibm.com/support/z-content-solutions/cloud-provisioning/)
```

Comment:
Was the intent here to have a clickable hyperlink?
When viewing the file on git Im still seeing [text] (link), this may be due to the space in between the two objects?
Googling got me this for what the syntax should look like for a clickable link

> [I'm an inline-style link](https://www.google.com)